website/_mechanics/basics/02-terminology.md

```diff
@@ -176,3 +176,21 @@ See [Slopesurfing](#slopesurfing)
 A tech that enters the wrong state when jumping, allowing an earlier wallrun.
 
 See [Raiserunning](#raiserunning)
+
+{:.no_toc}
+### Clip
+Certain configurations of geometry allow the player to pass through solid walls.
+
+{:.no_toc}
+### Magnet
+Jumping mid-air will "magnet" you to walls approximately within half of the width (TODO: fact check?) of your characters hitbox.
```

Comment:
Fact Check: Wall magnets work up to 41 - 42 units away, almost a full tile. differences due to symmetry bugs.

Reply:
```suggestion
Jumping mid-air will "magnet" you to walls up to 42 horizontal units away (almost a full tile).
```
Something like this?